crates/pallet-pot/src/lib.rs

```diff
@@ -33,8 +33,8 @@ pub type NegativeImbalanceOf<T, I = ()> =
 pub type CreditOf<T, I = ()> = Credit<<T as Config<I>>::AccountId, <T as Config<I>>::Currency>;
 
 /// The initial state of the pot, for use in genesis.
-#[cfg(feature = "std")]
-#[derive(Debug, serde::Serialize, serde::Deserialize)]
+#[derive(Debug)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
I think this could be submitted to `master` directly, this is irrelevant to the update

Reply:
https://github.com/paritytech/substrate/pull/13027

Reply:
Ah, got it!